app/Services/Category.php

```diff
@@ -2,6 +2,8 @@
 
 namespace App\Services;
 
+use App\Services;
+
 abstract class Category
```

Comment:
I know this has been sitting open for ages so I'm sorry to make one more request, but could you write tests for these two new methods?

Thanks so much!